### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by JustinDxx**
Refactored the code

### CODE REVIEW

**Comment by JustinDxx** [comment]
included change of endpoint for /id-documents due to did-web-resolution

File: src/self_description_creator.py [hunk 1]
```diff
@@ -295,10 +280,19 @@ def get_id_documents():
         app.logger.warning(error_msg)
         data = {"status": "failed", "error": error_msg}
         return data, 500
-
-
     
 
+@app.route("/id-documents/<request_uuid>/did.json", methods=["GET"])
```
Comment (robertschubert) [inline src/self_description_creator.py:285]: Hier braucht es noch eine Anpassung der OpenAPI und der postman collection

**Review by robertschubert — CHANGES_REQUESTED** [review]
approved.

**Review by robertschubert — APPROVED** [review]
approved

**Pull request merged**